policy/modules/system/init.te

```diff
@@ -2026,3 +2026,7 @@ optional_policy(`
         ccs_read_config(daemon)
     ')
  ')
+
+optional_policy(`
+	screen_exec(init_t)
```

Comment:
This would effect in screen running in the init_t context, that does not feel right.